products/memgraph/upcoming/how_to_guides/query-memgraph-programmatically.md

```diff
@@ -131,51 +133,134 @@ public class JavaQuickStart {
 The details about Javascript driver can be found on
 [GitHub](https://github.com/neo4j/neo4j-javascript-driver).
 
-Here is an example related to `Node.js`. Memgraph doesn't have integrated
-support for `WebSocket` which is required during the execution in any web
-browser. If you want to run `openCypher` queries from a web browser,
-[websockify](https://github.com/novnc/websockify) has to be up and running.
+Memgraph doesn't have integrated support for `WebSocket` which is required
+during the execution of Cypher commands in any web browser. If you want to
+run `openCypher` queries from a web browser,
+[websockify](https://github.com/novnc/websockify-js) has to be up and running.
 Requests from web browsers are wrapped into `WebSocket` messages, and a proxy
 is needed to handle the overhead. The proxy has to be configured to point out
 to Memgraph's Bolt port and web browser driver has to send requests to the
-proxy port.
+proxy port. Presented with Cypher language, the communication goes like this:
+
+```
+(:Browser:Javascript)-[:CONNECTS_TO]->(:Websockify { mode: "WS" })-[:PROXY_TO]->(:Memgraph { "encryption": "off" })
+```
+
+Proxy `Websockify` runs in unencrypted HTTP (ws://) mode by default which is a
+not secure Websocket protocol, so to match that, Memgraph needs to be running
+with encryption turned off. Check [Secure Sockets Layer (SSL)](#secure-sockets-layer)
+for more details on how to run Memgraph without encryption.
+
+The code snippet below outlines a basic usage example which executes a couple
+of elementary queries. Make sure to start Websockify to proxy queries to the
+database:
+
+```bash
+cd websockify
+npm install
+./websockify.js 9999 :7687
+```
+
+```html
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8"/>
+  <title>Javascript Browser Example | Memgraph</title>
+  <script src="https://cdn.jsdelivr.net/npm/neo4j-driver"></script>
+</head>
+<body>
+  <p>Check console for Cypher query outputs...</p>
+  <script>
+    const driver = neo4j.driver(
+      "bolt://localhost:9999",
+      neo4j.auth.basic("", ""),
+    );
+
+    (async function main() {
+      const session = driver.session();
+      
+      try {
+        await session.run("MATCH (n) DETACH DELETE n");
+        console.log("Database cleared.");
+
+        await session.run("CREATE (alice: Person {name: 'Alice', age: 22})");
+        console.log("Record created.");
+
+        const result = await session.run("MATCH (n) RETURN n");
+        console.log("Record matched:");
+        const alice = result.records[0].get("n");
+        console.log(alice.labels[0]);
+        console.log(alice.properties["name"]);
+
+      } catch (error) {
+        console.error(error);
+      } finally {
+        session.close();
+      }
+
+      driver.close();
+    })();
+  </script>
+</body>
+</html>
+```
+
+### Node.js Example {#node-js-example}
+
+The details about Javascript driver can be found on
```

Comment:
`The details about Javascript driver can be found on` --> `Details about the Javascript driver can be found on`